types/fs-base-ui/html_ui/JS/simvar.d.ts

```diff
@@ -0,0 +1,114 @@
+/// <reference path="./Types.d.ts" />
+/// <reference path="./Simplane.d.ts" />
+/// <reference path="../../../types.d.ts" />
+
+type NumberVarUnit = ("number" | "Number") | "position 32k" | ("SINT32") | ("bool" | "Bool" | "Boolean" | "boolean") | "Enum" | "lbs" | "kg" | ("Degrees" | "degree")
+    | "radians" | ("Percent" | "percent") | ("Feet" | "feet" | "feets" | "Feets") | "Volts" | "Amperes" | "Hertz" | "PSI" | "celsius" | "degree latitude"
+    | "degree longitude" | "Meters per second" | "Position" | ("Knots" | "knots") | "Seconds" | "seconds" | "kilograms per second" | "nautical miles" | "degrees"
+
+type TextVarUnit = "Text" | "string"
+
+type VarUnit = NumberVarUnit | TextVarUnit | LatLongAltVarUnit | LatLongAltPBHVarUnit
+    | PitchBankHeadingVarUnit | PID_STRUCTVarUnit | XYZVarUnit;
+
+type LatLongAltVarUnit = "latlonalt";
+type LatLongAltPBHVarUnit = "latlonaltpbh";
+type PitchBankHeadingVarUnit = "pbh";
+type PID_STRUCTVarUnit = "pid_struct";
+type XYZVarUnit = "xyz"
+type POIListVarUnit = "poilist";
+type FuelLevelsVarUnit = "FuelLevels";
+type GlassCockpitSettingsVarUnit = "GlassCockpitSettings";
+
+type SimVarBatchType = "string" | "number";
+
+declare global {
+    namespace SimVar {
+        class SimVarValue {
+            constructor(name?: string, unit?: VarUnit, type?: string);
+            name?: string;
```

Comment:
Should `__type` be listed here, since it's also listed for `RangeDisplay` and other displays?

Reply:
I'll remove it on `RangeDisplay` and the others. I was in doubt for a while, but I believe any `_` prefixed variable or `m_` variable was intended to be private.